.github/workflows/apengine-metadata-flexible.yaml

```diff
@@ -0,0 +1,67 @@
+name: apengine-metadata-flexible
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - 'apengine/metadata/flexible/**'
+  pull_request:
+    paths:
+    - 'apengine/metadata/flexible/**'
+  pull_request_target:
+    types: [labeled]
+  schedule:
+  - cron:  '0 0 * * 0'
+jobs:
+  test:
+    if: ${{ github.event.action != 'labeled' || github.event.label.name == 'actions:force-run' }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    permissions:
+      contents: 'write'
+      pull-requests: 'write'
+      id-token: 'write'
+    steps:
+    - uses: actions/checkout@v3.1.0
+      with:
+        ref: ${{github.event.pull_request.head.ref}}
+        repository: ${{github.event.pull_request.head.repo.full_name}}
+    - uses: 'google-github-actions/auth@v0.8.3'
+      with:
+        workload_identity_provider: 'projects/1046198160504/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions-provider'
+        service_account: 'kokoro-system-test@long-door-651.iam.gserviceaccount.com'
+        create_credentials_file: 'true'
+        access_token_lifetime: 600s
+    - uses: actions/setup-node@v3.5.1
+      with:
+        node-version: 16
+    - run: npm install
+      working-directory: apengine/metadata/flexible
+    - run: npm test
+      working-directory: apengine/metadata/flexible
+      env:
+        MOCHA_REPORTER_SUITENAME: apengine_metadata_flexible
+        MOCHA_REPORTER_OUTPUT: apengine_metadata_flexible_sponge_log.xml
+        MOCHA_REPORTER: xunit
+    - if: ${{ github.event.action == 'labeled' && github.event.label.name == 'actions:force-run' }}
+      uses: actions/github-script@v6
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        script: |
+          try {
+            await github.rest.issues.removeLabel({
+              name: 'actions:force-run',
+              owner: 'GoogleCloudPlatform',
+              repo: 'nodejs-docs-samples',
+              issue_number: context.payload.pull_request.number
+            });
+          } catch (e) {
+            if (!e.message.includes('Label does not exist')) {
+              throw e;
+            }
+          }
+    - if: ${{ github.event_name == 'schedule'}}
+      run: |
+        curl https://github.com/googleapis/repo-automation-bots/releases/download/flakybot-1.1.0/flakybot -o flakybot -s -L
+        chmod +x ./flakybot
+        ./flakybot --repo GoogleCloudPlatform/nodejs-docs-samples --commit_hash ${{github.sha}} --build_url https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
```

Comment:
**Issue:**
I think we need to redo the ["Add new samples" section from CONTRIBUTING.md](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this pull-request. 
You will need to:
1. Add "game-servers/snippets" to [/.github/workflows/workflows.json](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/workflows/workflows.json).
2. Run `node .github/workflows/generate.js`.
3. Only commit the `.github/workflows/game-servers-snippets.yaml` file. Do not commit the other files (from `.github/workflows`) that we created/modified by step 2.

Reply:
On it! 👍🏼